templates/partner/intel.html

```diff
@@ -0,0 +1,645 @@
+            Easily develop and deploy your machine learning models using Charmed Kubeflow, on any infrastructure including support for Intel GPUs.
+          </p>
+        </div>
+        <div class="p-equal-height-row__item">
+          <p>
+            <a href="/ai/what-is-kubeflow"
+               aria-label="Learn more about Charmed Kubeflow">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/661822e8-canonical.png",
+                        alt="Canonical",
+                        width="852",
+                        height="480",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--muted u-hide--medium" />
+          <p>
+            The Canonical Data Science Stack is an out-of-the-box solution to develop and optimize models, designed to run on any Intel workstation without spending time on tooling setup.
+          </p>
+        </div>
+        <div class="p-equal-height-row__item">
+          <p>
+            <a href="/ai/data-science"
+               aria-label="Learn more about Canonical data-science">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Explore Ubuntu-certified Intel Hardware backed by Ubuntu Pro</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <p>
+            Certified hardware is rigorously tested to provide the best Ubuntu experience. Even after certification, Canonical executes continuous regression testing throughout the Ubuntu release life cycle on the full fleet of certified devices and offers 12 years of support and security updates with <a href="/pro">Ubuntu Pro</a>.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="p-equal-height-row">
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item u-hide--small u-hide--medium">
+          <div class="p-image-container">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/01ffd135-laptops.png",
+            alt="Laptops",
+            width="852",
+            height="1278",
+            hi_def=True,
+            attrs={"class": "p-image-container__image"},
+            loading="auto|lazy"
+            ) | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">
```

Comment:
Should the links here be removed? I don't see them on the copy doc and also saw this comment thread: https://docs.google.com/document/d/1GstP9FfZfbXO81YcX-9oCJdP9pamlgpl2kKkSqlrWFE/edit?disco=AAABaVZgdN0